database/migrations/2019_06_18_113551_create_appointments_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAppointmentsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+
+    protected $primaryKey = 'appointment_id';
```

Comment:
We shouldn't need this. Add the `id` into the migration as so `$table->increments('id');`